Backend/app/routers/service_accounts.py

```
@@ -0,0 +1,237 @@
+def admin_refresh_all_expired_tokens(
+    session: SessionDep,
+    max_count: int = 100
+):
+    try:
+        stats = batch_refresh_expired_tokens(session, max_count=max_count)
+        return {
+            "success": True,
+            "stats": stats,
+            "message": f"Refreshed {stats['oauth_connections']['success'] + stats['service_accounts']['success']} tokens"
+        }
+    except Exception as e:
```

Ruff (0.14.6)

Do not catch blind exception: `Exception`

(BLE001)